catroid/src/main/res/values/strings.xml

```diff
@@ -761,6 +761,11 @@
     <string name="project_select_screen_title">Screen:</string>
     <!--  -->
 
+    <!-- NewMergedProjectDialog -->
+    <string name="new_merge_project_dialog_title">Merged project</string>
+    <string name="temp_merge_text">Merging</string>
```

Comment:
Please don't call it `temp_*`. We have a lot of "temporary" fixes and functions that turned out to be permanent 😅